tree-sitter/test-codegen/test-codegen.hs

```diff
@@ -39,10 +39,10 @@ prop_initUpper = property $ do
   (y:ys) <- pure $ initUpper (x:xs)
   when (isLower x) (assert (isUpper y))
 
-prop_mapOperator :: Property
-prop_mapOperator = property $ do
+prop_escapePunct :: Property
+prop_escapePunct = property $ do
```

Comment:
you've been Punct

![image](https://user-images.githubusercontent.com/875834/60460713-810d7d00-9c12-11e9-87fb-26b4f7b916d8.png)